CMakeLists.txt

```diff
@@ -69,6 +76,8 @@ include(cmake/libbpf.cmake)
 # install frida
 include(cmake/frida.cmake)
 
+# Set default value for BPFTIME_LLVM_JIT to ON
+option(BPFTIME_LLVM_JIT "Use LLVM as jit backend." ON)
```

Comment:
Maybe it would be better if you only change the default value of this option, and remain other parts of cmake untouched?

Reply:
Got it. Fixed it according to your suggestion. It works!